components/expenses/ExpenseFormPayeeStep.js

```diff
@@ -233,7 +233,6 @@ const ExpenseFormPayeeStep = ({
               setLocationFromPayee(formik, payee);
             }
           }}
-          limit={5}
```

Comment:
What is the limit if we remove the limit?

Reply:
The limit becomes 20 which is the default for `CollectivePickerAsync`;

https://github.com/opencollective/opencollective-frontend/blob/565fdcb209a3b6c02f9eada3918efa7d4855bbf1/components/CollectivePickerAsync.js#L178